modules/react/skeleton/stories/Skeleton.stories.mdx

```diff
@@ -55,72 +55,28 @@ ensure the Skeleton components are visible.
 
 <ExampleCodeBlock code={Color} />
 
-## Components
-
-### Skeleton
-
-#### Usage
-
-Skeleton subcomponents **must** be wrapped by the `Skeleton` container component.
-
-```tsx
-<Skeleton>
-  <Skeleton.Header />
-  <Skeleton.Text />
-</Skeleton>
-```
-
-`Skeleton` places its children in a container element with `aria-hidden` set to `true` and announces
-itself using a visually hidden element.
-
-#### Props
-
-Undocumented props are spread to the outermost element rendered by `Skeleton`.
-
-<ArgsTable of={Skeleton} />
-
-### Skeleton.Header
-
-#### Usage
-
-`Skeleton.Header` renders a placeholder for header content such as headings.
-
-<ExampleCodeBlock code={Header} />
-
-#### Props
-
-Undocumented props are spread to the outermost element rendered by `Skeleton.Header`.
-
-<ArgsTable of={Skeleton.Header} />
-
-### Skeleton.Text
-
-#### Usage
+### Text
 
 `Skeleton.Text` renders a placeholder for text content such as paragraphs. Each placeholder line has
 a width of `100%` and a fixed height of `21px`, with the last line having a width of `60%` if there
 are multiple lines.
 
 <ExampleCodeBlock code={Text} />
 
-#### Props
-
-Undocumented props are spread to the outermost element rendered by `Skeleton.Text`.
+### Header
 
-<ArgsTable of={Skeleton.Text} />
+`Skeleton.Header` renders a placeholder for header content such as headings.
 
-### Skeleton.Shape
+<ExampleCodeBlock code={Header} />
 
-#### Usage
+### Shape
 
 `Skeleton.Shape` renders a placeholder for graphic elements such as icons, avatars and small images.
 Set the `height`, `width`, and `borderRadius` props of the `Skeleton.Shape` to create various
 rectangular and circular shapes.
 
 <ExampleCodeBlock code={Shape} />
 
-#### Props
-
-Undocumented props are spread to the outermost element rendered by `Skeleton.Shape`.
+## Component API
 
-<ArgsTable of={Skeleton.Shape} />
+<SymbolDoc name="Skeleton" hideDescription />
```

Comment:
I see `hideDescription` here but I see what look like descriptions rendering for each subcomponent above their Props section, so each description appears twice on the page instead of once like before.

Reply:
That is expected behavior. The `<SymbolDoc>` component will render a description by default:

```tsx
// SymbolDoc
<StyledSymbolDoc {...elemProps}>
  <HeadingLevelContext.Provider value={headingStart}>
    {!hideDescription && doc && <MdxJSToJSX>{doc.description}</MdxJSToJSX>}
    {contents}
  </HeadingLevelContext.Provider>
</StyledSymbolDoc>
```

But the `enhancedComponent` widget will also render a description because it is recursive and also renders the name of the component (and subcomponents)
```tsx
/// enhancedComponent.tsx
<ParentComponentNameContext.Provider value="">
  <ParentComponentJSDocContext.Provider value={defaultJSDoc}>
    <MdxJSToJSX>{parentComponentJSDoc.description || doc?.description || ''}</MdxJSToJSX>
  </ParentComponentJSDocContext.Provider>
</ParentComponentNameContext.Provider>
```

Reply:
I add `hideDescription` to all enhanced component symbols for this reason. I don't know enough about the context to know if a description should be rendered, so it is left up to the user.